docs/web-apps/automated-testing/selenium/selenium4.md

```diff
@@ -30,15 +33,15 @@ Below are features new to Selenium 4 that may impact your tests.
 </tr>
 <tr>
 <td>Force W3C Usage</td>
-<td>Legacy JSON wire protocol support deprecated → uses the W3C WebDriver standard by default under the hood Force W3C usage (except Java). You can no longer use outdated DesiredCapabilities. No longer backwards compatible.</td>
+<td>Selenium 4 deprecates support for legacy JSON wire protocol and its static <code>DesiredCapabilities</code> browser methods, replacing it with <a href="/dev/w3c-webdriver-capabilities">W3C WebDriver-compliant protocol</a> and the use of <code>browserOptions</code>. You must completely switch from using JWP to W3C; Selenium will no longer be backwards-compatible. The W3C standard will be supported by default under the hood. </td>
 </tr>
 <tr>
 <td>Relative Locators</td>
-<td>A new way of locating elements by using more natural language friendly terms, such as “above”, “below”, “left of”, “right of”, and “near”. In general, you could think about them as a way to locate elements based on the visual placement on the page. <a href="https://github.com/saucelabs-training/demo-java/blob/main/training-sessions/selenium4/src/test/java/com/saucelabs/selenium4/new_/RelativeLocatorsTest.java">Code example</a>.</td>
+<td>New way of locating elements based on the visual placement on the page. Relative locators use more natural language friendly terms, such as “above”, “below”, “left of”, “right of”, and “near”. <a href="https://github.com/saucelabs-training/demo-java/blob/main/training-sessions/selenium4/src/test/java/com/saucelabs/selenium4/new_/RelativeLocatorsTest.java">See a code example</a>.</td>
 </tr>
 <tr>
 <td>New Window and Tab Utilities</td>
-<td>New command to help users create a new tab or a new window. Better window and tabs managing <a href="https://github.com/saucelabs-training/demo-java/blob/main/training-sessions/selenium4/src/test/java/com/saucelabs/selenium4/new_/NewWindowSauceTest.java">Code example</a>.</td>
+<td>New command that allows you to create and manage new tabs and new windows. <a href="https://github.com/saucelabs-training/demo-java/blob/main/training-sessions/selenium4/src/test/java/com/saucelabs/selenium4/new_/NewWindowSauceTest.java">See a code example</a>.</td>
 </tr>
 <tr>
 <td>Print Page as PDF</td>
```

Comment:
Missing an example below!

Reply:
Hi @titusfortner , for the **Print Page as PDF**, do you have an example in GitHub? the only ones I could find are in Diego's doc https://docs.google.com/document/d/1IZ-Pk2XjU5gEX1heRjjLq-E0ee-ezF8GXhqc5bWSoFM/edit